Project EFT/Data Classes/AnswerSubmission.cs

```diff
@@ -16,5 +17,15 @@ public AnswerSubmission(string content, DateTime submissionDate, int id, bool is
             IsCorrect = isCorrect;
             ProblemId = problemID;
         }
+
+        //for posterity sake, this used to be an override of Equals, which required a custom GetHashCode function as well...it was never committed, for good reason...
+        //I will allow you to picture what that monstrosity looked like
+        public bool IsEqual(AnswerSubmission subs)
+        {
+            return (this.Content.Equals(subs.Content)
+                    && this.SubmissionDate.Equals(subs.SubmissionDate)
+                    && this.UserID == subs.UserID && this.IsCorrect == subs.IsCorrect
+                    && this.ProblemId == subs.ProblemId);
+        }
```

Comment:
should probably override a virtual method in parent class to disallow comparing AnswerSubmission with Submission, unless that is desired

Reply:
I tried making the Submission classes ```IsEqual(Submission subs)``` virtual, but then since the AnswerSubmission's IsEqual is defined as ```IsEqual(AnswerSubmission subs)``` it does not override the method in the parent class, since they have different parameters input.   https://stackoverflow.com/questions/27547122/c-sharp-override-with-different-parameters

I left it as is for now.